README.md

```diff
@@ -2,7 +2,7 @@
 
 This repo houses the Go SDK for integrating with the [Extism](https://extism.org/) runtime. Install this library into your host Go applications to run Extism plugins.
 
-Join the [Discord](https://discord.gg/EGTV8Pxs) and chat with us!
+Join the [Extism Discord](https://discord.gg/UsNqcTTa9P) and chat with us!
```

Comment:
Why does the url need to change? we also have `https://extism.org/discord` btw

Reply:
The old link is dead for me: 
![image](https://github.com/extism/go-sdk/assets/2080147/dc4b643b-838d-48d4-a4c9-eb0da9f5e811)

Does it work for you?